tests/airgapped/README.md

```diff
@@ -59,7 +59,9 @@ Devs are urged to instead define their own `images.txt` file with the images
 they'd like to be loaded during tests.
 
 ```bash
-./scripts/airgapped/get-all-images.sh releases/1.8/stable/kubeflow/bundle.yaml > images-all.txt
+python3 scripts/get-all-images.py \
+    releases/1.8/stable/kubeflow/bundle.yaml \
```

Comment:
I think we should make the example on our latest release i.e. 1.9

Reply:
Or leave it generic so we do not have to come back and change it every time?

Reply:
I like keeping this with sane values, so they can be copy pasted from README, but at the same time would like to do for 1.9.

But to properly update for 1.9 we'll need to also update the file that includes the airgap-testing images. So this should be done in a follow-up PR

Reply:
Manos filed #1051 , can you comment this there?

Reply:
Done!